delivery/templates/googlemaps.ftl

```diff
@@ -33,21 +33,23 @@
       content: '';
       position: absolute;
       display: inline-block;
-      width: ${contentModel.width_s}px;
-      height: ${contentModel.height_s}px;
+      width: ${contentModel.width_s};
+      height: ${contentModel.height_s};
     }
   </style>
 </#if>
 
-<@crafter.componentRootTag class="craftercms-googlemaps-plugin-container">
+<@crafter.componentRootTag>
   <#if apiKey?has_content>
-    <@crafter.iframe
-      allowfullscreen="true"
-      width="${contentModel.width_s}"
-      height="${contentModel.height_s}"
-      frameborder="0" style="border:0"
-      src="https://www.google.com/maps/embed/v1/place?key=${apiKey}&${params?join('&')}"
-    />
+    <div class="craftercms-googlemaps-plugin-container">
+      <@crafter.iframe
+        allowfullscreen="true"
+        width="${contentModel.width_s}"
+        height="${contentModel.height_s}"
+        frameborder="0" style="border:0"
+        src="https://www.google.com/maps/embed/v1/place?key=${apiKey}&${params?join('&')}"
+      />
```

Comment:
Should either have a field or not be a @crafter element

Reply:
Correct, if doesn't represent a field in the component, changing it to be an iframe element